e2e/__projects__/tsconfig/test.js

```diff
@@ -0,0 +1,54 @@
+import { getTSConfig } from 'vue-jest/lib/utils'
```

Comment:
This directory is intended for e2e tests. Can you move these tests into a unit test file: /lib/utils.spec.js, and create a `test:unit` script that runs files with a spec extension that aren't in /e2e